content/v2.1/helpers/overview.md

```diff
@@ -3,8 +3,231 @@ title: Overview
 order: 10
 ---
 
-Hanami provides a range of standard helpers for you to use in your views.
+Hanami provides a range of standard helpers for you to use in your [views](/v2.1/views/overview/).
 
 You can read more about where helpers fit within your views in the [view helpers guide](/v2.1/views/helpers/).
 
 To learn more about Hanami's standard helpers, read the sections within this guide.
+
+## Tag Helper
+
+This helper makes accessible all the HTML-related tags.
+
+### Usage
+
+Here is how you can use it **in templates**:
+
+```ruby
+<%= tag.div(id: "el") do %>
+  <p>Template content can be mixed in.</p>
+  <%= tag.p("Also nested tag builders.") %>
+<% end %>
+```
+
+This will render:
+
+```html
+<div id="el">
+  <p>Template content can be mixed in.</p>
+  <p>Also nested tag builders.</p>
+</div>
+```
+
+[In parts](/v2.1/views/parts), you can access your helpers via the `helpers` object.
+
+Given you have a view exposure defined:
+
+```ruby
+# app/views/books/show.rb
+
+module Bookshelf
+  module Views
+    module Books
+      class Show < Bookshelf::View
+        expose :fancy_header do
```

Comment:
@swilgosz Given parts are a new concept. Readers may have a hard time wrapping their heads around them.
For me, this is an advanced usage of Parts.

What if we replace `:fancy_header` with a simple `:book`?

```ruby
expose :book do
  Book.new(title: "Hanami")
end
```

Then the part will be about `Book`, not `FancyHeader`.

```ruby
def content
  helpers.tag.h1(value.title)
end
```

Reply:
I like this simplification, thanks!

Reply:
Done!